internal/gpustats/gpustats.go

```diff
@@ -6,72 +6,51 @@ import (
 	"github.com/gpuctl/gpuctl/internal/uplink"
 )
 
-type UncontextualGPUStats struct {
-	MemoryUtilisation float64 `json:"memory_util"`
-	GPUUtilisation    float64 `json:"gpu_util"`
-	MemoryUsed        float64 `json:"memory_used"`
-	FanSpeed          float64 `json:"fan_speed"`
-	Temp              float64 `json:"gpu_temp"`
-}
+var (
+	SampleAdditionError = errors.New("two packets with different contexts cannot be aggregated using Add, consider using UncontextualAdd")
+)
 
-// Combine two GPUStats instances into one.
-func Add(l, r uplink.GPUStats) (uplink.GPUStats, error) {
-	if r.Name != l.Name || r.Brand != l.Brand || r.DriverVersion != l.DriverVersion || r.MemoryTotal != l.MemoryTotal {
-		// TODO: Expose this error publicly.
-		return uplink.GPUStats{}, errors.New("two packets with different contexts cannot be aggregated using Add, consider using UncontextualAdd")
+// Combine two GPUStatSamples instances into one.
```

Comment:
NIT: What, what are the sematics of this function suposed to be? When would it be used?

I'm pritty sure adding 2 70C gpu's should make a 140C gpu.

Probably best to remove at some point, as I don't think it get's used.

Reply:
I believe @Euphoride added it back in (https://github.com/gpuctl/gpuctl/commit/325072c55246cb373d2a03f7f319d330193d05f4), so this isn't new, but my understanding is with the satellite locally storing a backlog of status packets and only sending only every so often (#68), we want a way to aggregate all the packets together (by summing so we can divide and average, etc.) into one